### PR TITLE
Release v0.25.0

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by knanao** [comment]
🙌 
/lgtm

**Comment by nakabonne** [comment]
I am happy to be able to face this fancy release.
/approve

**Comment by nghialv** [comment]
> I am happy to be able to face this fancy release. /approve

@nakabonne You made it. Thank you for your great contributions!

File: docs/content/en/blog/releases/v0.25.0.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+### New Features
+
+* The event list is now available on the web console ([#3146](https://github.com/pipe-cd/pipecd/pull/3146))
+
+### Notable Changes
+
+* Make it possible to customize the variant label of Kubernetes application ([#3179](https://github.com/pipe-cd/pipecd/pull/3179))
+* Ensure that the Pipeline sync works normally even when the config file name was changed ([#3148](https://github.com/pipe-cd/pipecd/pull/3148))
+* Fix the bug of plan-preview action that caused preview comments not to be hidden ([#3157](https://github.com/pipe-cd/pipecd/pull/3157))
```
Comment (khanhtc1202) [inline docs/content/en/blog/releases/v0.25.0.md:19]: do we need this in notable change section? looks like the below is enough 👀

**Comment by khanhtc1202** [comment]
oops, too late 😅

File: docs/content/en/blog/releases/v0.25.0.md (hunk 1 above)
Comment (nghialv) [reply docs/content/en/blog/releases/v0.25.0.md:19]: Ok. Let me update the release note later.

**Pull request merged**